LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2017 Pablo Mansanet (Absolute Software Ltd)
```

Comment:
I asked Matthew and he told me to put both my name and the company name, but I'm not sure that's valid?